src/array/boolean/mutable.rs

```diff
@@ -104,6 +104,69 @@ impl MutableBooleanArray {
         }
     }
 
+    /// Extends the [`MutableBooleanArray`] from an iterator of values of trusted len.
+    /// This differs from `extend_trusted_len` which accepts in iterator of optional values.
+    #[inline]
+    pub fn extend_trusted_len_values<I>(&mut self, iterator: I)
+    where
+        I: TrustedLen<Item = bool>,
+    {
+        unsafe { self.extend_trusted_len_values_unchecked(iterator) }
```

Comment:
```suggestion
        /// Safety: `I` is `TrustedLen`
        unsafe { self.extend_trusted_len_values_unchecked(iterator) }
```

Reply:
The Safety comment was added to every place inside the file where we use unsafe blocks, but `///` wasn't possible